### PR TITLE
Encontrou diferencas e corrige as mesmas.

### PR DESCRIPTION
**Pull request by Rockabilico**
@casey-collab

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, you there?

File: portugues/colabora/comdiferencas/comdiferencas.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 Você encontra os dois commits que são o divisores de água entre o aparecimento do bug que tanto lhe incomoda. 
 Você então:
 
+[Compara e corrige os erros para que tudo funcione.](../corrigedif/corrigedif.md)
```
Comment (casey-collab) [inline portugues/colabora/comdiferencas/comdiferencas.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: portugues/colabora/comdiferencas/corrigedif/corrigedif.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Salvou o dia de muitos utilizadores corrigindo um bug! No entanto há que submeter essa alteraçao.
```
Comment (casey-collab) [inline portugues/colabora/comdiferencas/corrigedif/corrigedif.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Unfortunately, this pull request conflicts with another change. Could you merge the master branch of the original repository into your branch, resolve the merge conflicts, and update the pull request?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**